Add svg node to enhanced grid checkbox. Couldn't find a good way to add this in umc/widgets/MultiSelect
--- dojox/grid/enhanced/plugins/IndirectSelection.js.orig	2020-11-30 14:51:14.801899196 +0100
+++ dojox/grid/enhanced/plugins/IndirectSelection.js	2020-11-30 14:54:55.827902449 +0100
@@ -90,6 +90,7 @@
 				"name = '" + _this.grid.id + "_rowSelector' class = '" + clazz + "' ",
 				"role = " + _this.inputType.toLowerCase() + " aria-checked = '" + checked + "' aria-disabled = '" + disabled +
 				"' aria-label = '" + string.substitute(_this.grid._nls["indirectSelection" + _this.inputType], [rowIndex + 1]) + "'>",
+				"<svg class='featherIcon icon-check' xmlns='http://www.w3.org/2000/svg'><use xlink:href='/univention/js/dijit/themes/umc/images/feather-sprite.svg#check'/></svg>",
 				"<span class = '" + _this.statusTextClass + "'>" + (checked ? _this.checkedText : _this.unCheckedText) + "</span>",
 				"</div>"].join("");
 	},
@@ -496,7 +497,8 @@
 		var selector = headerCellNode.appendChild(html.create("div", {
 			'aria-label': g._nls["selectAll"],
 			"tabindex": -1, "id": g.id + "_rowSelector_-1", "class": this.baseClass, "role": "checkbox",
-			"innerHTML": "<span class = '" + this.statusTextClass +
+			"innerHTML": "<svg class='featherIcon icon-check' xmlns='http://www.w3.org/2000/svg'><use xlink:href='/univention/js/dijit/themes/umc/images/feather-sprite.svg#check'/></svg>" +
+				"<span class = '" + this.statusTextClass +
 				"'></span><span style='height: 0; width: 0; overflow: hidden; display: block;'>" +
 				g._nls["selectAll"] + "</span>"
 		}));
